src/node/settings.ts

```diff
@@ -0,0 +1,72 @@
+const fsPromises = require('fs/promises');
+const fs = require('fs');
+const path = require('path');
+
+import type { Pin } from '../types';
+import { getSettingsDirectory } from './constants';
```

Comment:
add `getSettingsFile` as well

Reply:
this is not needed now, right? or correct me if am wrong

Reply:
yes, let's have in one single place not hardcoded in code